src/js/helpers/container/index.js

```diff
@@ -1,10 +1,10 @@
-import * as d3 from 'd3';
+import d3 from 'd3';
 import deepmerge from 'deepmerge';
 
-import momentum from '../momentum';
-import Models from '../../models';
-import ContainerZoom from './zoom';
-import ContextMenu from '../contextMenu';
+import momentum from '../momentum/index.js';
+import Models from '../../models.js';
+import ContainerZoom from './zoom.js';
+import ContextMenu from '../contextMenu/index.js';
```

Comment:
What is the reason to import modules from `index` files with file extension `.js`?

Reply:
This is explained in the associated commit message.